multipaz-compose/src/commonMain/kotlin/org/multipaz/compose/camera/FrameProcessor.kt

```diff
@@ -0,0 +1,9 @@
+package org.multipaz.compose.camera
+
+/**
+ * WIP. Common interface fo various camera frame data processors used by VideoOutputDelegate platform implementations.
+ */
+expect interface FrameProcessor {
+    fun detectFace(frameData: ByteArray): Boolean
+    fun matchFace(frameData: ByteArray): String?
+}
```

Comment:
I don't think the interface is going to look like this so I would leave out all the FrameProcessor things for now, especially b/c we're about to do a release of this code. Including the `frameProcessor: FrameProcessor` parameter to the `Camera` composable.